template/content/clients.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+var TChannel = require('tchannel');
+var HyperbahnClient = require('tchannel/hyperbahn');
+var fs = require('fs');
+var path = require('path');
+var Ringpop = require('ringpop');
+var Logtron = require('logtron');
+var Statsd = require('uber-statsd-client');
+var os = require('os');
+var process = require('process');
+var myLocalIp = require('my-local-ip');
+var Uncaught = require('uncaught-exception');
+
+var thriftFile = fs.readFileSync(
+    path.join(__dirname, 'thrift', 'service.thrift'), 'utf8'
+);
+
+var SERVICE_NAME = 'my-service';
+
+/* TODO:
+    - process reporter
+    - heap dump
+    - repl
+*/
+module.exports = ApplicationClients;
+
+function ApplicationClients(config, options) {
```

Comment:
This ctor is already busy. It'll only get busier once they add their client dependencies. Have you thought about ways to decompose this file into smaller bits? One can go too far with this idea too, so I understand if you want to keep it monolithic and simple.